tgui-next/packages/tgui/interfaces/MedicalKiosk.js

```diff
@@ -0,0 +1,153 @@
+import { Fragment } from 'inferno';
+import { act } from '../byond';
+import { AnimatedNumber, Box, Button, LabeledList, ProgressBar, Section } from '../components';
+
+export const MedicalKiosk = props => {
+  const { state } = props;
+  const { config, data } = state;
+  const { ref } = config;
+ 
+  return (
+    <Fragment>
+      <Section title="Health Kiosk"
+        textAlign="center">
+        <Box as="span" m={1} textAlign="center">
```

Comment:
`as="span"` likely not needed here, if it needs to display inline just add the `inline` prop.

Reply:
It's technically same, so not worth changing.

Reply:
It's more about enforcing proper practice early on so that people don't copypaste bad practice ad infinitum. This happens a lot.